personal-portfolio/js/validation.js

```diff
@@ -0,0 +1,20 @@
+const formValidation = document.querySelector('#form');
+const emailControl = document.querySelector('#email-address');
+
+const alertMessage = document.createElement('p');
+alertMessage.classList.add('error-Message');
+alertMessage.textContent = 'Your email should only contain lowercase letters.';
+
+const errorEmail = document.querySelector('.email-msg');
+errorEmail.appendChild(alertMessage);
+
+function validationFunct (e) {
+  if (emailControl.value !== emailControl.value.toLowerCase()) {
+    errorEmail.style.display = 'block';
+    e.preventDefault();
+  } else {
+    errorEmail.style.display = 'none';
+  }
+}
+
+formValidation.addEventListener('submit', validationFunct);
```

Comment:
**Nice job on adding the validation @mprotic123**

- [ ] You still need to fix the validation form, the form can submit a uppercase input. [see the requirement](https://user-images.githubusercontent.com/61562626/128514940-87c15ba9-58d5-4f81-a39c-fca43d77841f.jpeg)

Reply:
Thank you for the review @dasileker. I'm not sure that i understand what you are trying to say because the validation works totally fine on my side. Can you please tell me what is the problem?

Reply:
Hello, let me explain more your validation form is working also for the uppercase, but it should not, it should render an error that uppercases are not allowed

Reply:
https://user-images.githubusercontent.com/82707385/128518406-22d61392-df82-4f78-b110-496163ccc89b.mp4

Excuse me sir but I don't think I know what are you trying to say?

Reply:
Okay, it displays In the btn you can send it for another review but I will highly recommend you that the error message should separate, it will be nice if it display as an alert.

Reply:
Yes, it would look much better but project requirements are saying that message should display in the button.